src/tool-core/models/tool-core/architecture/classes/Attribute/Attribute.xtuml

```diff
@@ -54,6 +54,7 @@ INSERT INTO O_TFR
 	'select any accessor related by self->AttributeAccessor[R4510.''value accessed through'']
  where ( selected.accessor_type == AttributeAccessorType::GETTER );
 gettername = accessor.getName();
+key = STRING::substr( s:self.name, begin:2, end:-1 );
```

Comment:
While we both remember why this here right now, I think this line deserves a comment.